solr/CHANGES.txt

```diff
@@ -163,6 +163,8 @@ Other Changes
 
 * SOLR-17142: Fix Gradle build sometimes gives spurious "unreferenced license file" warnings. (Uwe Schindler)
 
+* SOLR-17448: Audit usage of ExecutorService#submit in Solr codebase (Andrey Bozhko)
```

Comment:
Could you reword this into its impact instead of its technical detail?  Like maybe "fixed inadvertent suppression of exceptions across the codebase due to ...""

Reply:
Updated